doc/api/perf_hooks.md

```diff
@@ -201,6 +201,25 @@ The value may be one of:
 * `perf_hooks.constants.NODE_PERFORMANCE_GC_INCREMENTAL`
 * `perf_hooks.constants.NODE_PERFORMANCE_GC_WEAKCB`
 
+### performanceEntry.flags
+<!-- YAML
+added v?
```

Comment:
What version I should point here?

Reply:
`REPLACEME`, i.e.
```suggestion
added: REPLACEME
```

When this PR is released the releaser will replace this placeholder with the actual version number.